src/backend/marsha/core/templates/core/lti.html

```diff
@@ -9,7 +9,7 @@
     {% endif %}
   </head>
   <body>
-    <div class="marsha-frontend-data"{% if jwt_token %} data-jwt="{{ jwt_token }}"{% endif %} data-state="{{ state }}"></div>
+    <div class="marsha-frontend-data"{% if jwt_token %} data-jwt="{{ jwt_token }}"{% endif %} data-state="{{ state }}" data-editable="{{ editable }}"></div>
```

Comment:
I have two issues here:
- `editable` in `{{ editable }}` should be JSON so we don't have to manually cast python types into javascript (`"true"` and `"false"` are valid JSON)
- IMO `editable` on `appData` should be `isEditable` or at lease `is_editable`. I'm not sure this is easy to achieve with data-attributes. Maybe the whole payload of this div would be better as `data="{whatever json}"`. WDYT?

Reply:
WDYT @sampaccoud ? instead of having multiple `data-attributes` having just one containing a json ? This is the kind of refacto we can do for a major release.

Reply:
I'm good. It will also simplify parsing on the frontend... so let's go! Not a breaking change anyway because it is internal between the frontend and backend...

Reply:
👍

Reply:
Modification made, you should review all the PR again.